main/WebServer.cpp

```diff
@@ -1691,7 +1691,7 @@ namespace http {
 			std::string variablename = request::findValue(&req, "vname");
 			std::string variablevalue = request::findValue(&req, "vvalue");
 			std::string variabletype = request::findValue(&req, "vtype");
-			if ((variablename == "") || (variablevalue == "") || (variabletype == ""))
+			if ((variablename == "") || (variabletype == "") || (variablevalue == "" && variabletype != "2"))
```

Comment:
Thanks!

To keep it readable (also the below patch) could you change to code like:

if (
----(variablename == "") || 
----(variabletype == "") || 
----( (variablevalue == "") && (variabletype != "2") )
) {
   .....
}